docs/index.rst

```diff
@@ -1,106 +1,79 @@
-  provides encryption of user data.
-
-* Scalability:
-
-  Toil can easily handle workflows concurrently using hundreds of nodes and
-  thousands of cores.
-
-* Robustness:
-
-  Toil workflows support arbitrary worker and leader failure, with strong
-  check-pointing that always allows resumption.
-
-* Efficiency:
+Toil is an open-source pure-Python workflow engine that lets people write better
+pipelines. You can:
 
-  Caching, fine grained, per task, resource requirement specifications, and
-  support for the AWS spot market mean workflows can be executed with little
-  waste.
+* Write your workflows in `Common Workflow Language`_ (CWL),
+* Run workflows on your laptop or on huge commercial clouds such as
+  `Amazon Web Services`_ (including the `spot market`_), `Microsoft Azure`_,
+  `OpenStack`_, and `Google Compute Engine`_,
+* Take advantage of high-performance computing environments with batch systems
+  like `GridEngine`_, `Apache Mesos`_, and `Parasol`_,
+* Run workflows concurrently at scale using hundreds of nodes and thousands of cores,
+* Execute workflows efficiently with caching and resource requirement specifications, and
+* Easily link databases and services
 
-* Declarative and dynamic workflow creation:
-
-  Workflows can be declared statically, but new jobs can be added dynamically
-  during execution within any existing job, allowing arbitrarily complex
-  workflow graphs with millions of jobs within them.
-
-* Support for databases and services:
-
-  For example, Apache Spark clusters can be created quickly and easily
-  integrated within a toil workflow as a service, with precisely defined time
-  start and end times that fits with the flow of other jobs in the workflow.
-
-* Open Source: An Apache license allows unrestricted use, incorporation and modification.
+Toil is just as good if not better than sliced bread, unless you are hungry, in
+which case we recommend seeking another solution.
+Click `here`_ to learn more about Toil and what it can do, or jump in and skip
+to :ref:`Installation`. (You can also join us on `GitHub`_ or `Gitter`_.)
 
 .. _GridEngine: http://gridscheduler.sourceforge.net/
-.. _Parasol: https://users.soe.ucsc.edu/~donnak/eng/parasol.htm
+.. _Parasol: http://genecats.soe.ucsc.edu/eng/parasol.html
 .. _Apache Mesos: http://mesos.apache.org/
 .. _spot market: https://aws.amazon.com/ec2/spot/
 .. _Microsoft Azure: https://azure.microsoft.com
 .. _Amazon Web Services: https://aws.amazon.com/
 .. _Google Compute Engine: https://cloud.google.com/compute/
 .. _OpenStack: https://www.openstack.org/
-.. _CWL: http://commonwl.org
-.. _Common Workflow Language: http://commonwl.org
-.. _WDL: https://github.com/broadinstitute/wdl
-.. _Workflow Description Language: https://github.com/broadinstitute/wdl
+.. _GitHub: https://github.com/BD2KGenomics/toil
+.. _Gitter: https://gitter.im/bd2k-genomics-toil/Lobby
+.. _here: https://bd2kgenomics.github.io/toil/
+
+Getting Started
+~~~~~~~~~~~~~~~
```

Comment:
I think it should at least stay in place until we have a code example to replace it with. (I think it's valuable, though, in that it's a deeper table of contents than the sidebar has, which allows skipping to a specific section much more quickly.)